src/arch/ArchHooks/ArchHooks.h

```diff
@@ -2,6 +2,7 @@
 #define ARCH_HOOKS_H
 
 #include <chrono>
+#include "Etterna/Globals/global.h"
```

Comment:
What's this for?

Reply:
tested this locally; will not compile without it

Reply:
I understand that, but it seems like the proper way to do this is to `#include <string>` and whatever else is needed instead.

Reply:
I suppose it's slightly out of scope of this PR, but this definitely shouldn't be needed compared to a more granular `#include`.

Reply:
replacing with `#include <string>` does solve this on Windows

Reply:
Updated, and will let CI check if it breaks other os compilation.